db/migrate/20200221215702_add_signed_response_message_requested_to_service_provider.rb

```diff
@@ -0,0 +1,10 @@
+class AddSignedResponseMessageRequestedToServiceProvider < ActiveRecord::Migration[5.1]
+  def up
+    add_column :service_providers, :signed_response_message_requested, :boolean
+    change_column_default :service_providers, :signed_response_message_requested, false
```

Comment:
can this be done with like a `default: false` in the previous line?

Reply:
I had that originally, but then am following the advice of the `strong_migrations` gem:

https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value

Reply:
Neat, TIL, thanks!